lib/routes/universities/cug/gcxy/index.js

```diff
@@ -18,48 +19,40 @@ module.exports = async (ctx) => {
             url: '/index/tzgg.htm',
         },
         {
-            name: '党建工作',
-            url: '/index/djgz.htm',
+            name: '党建新闻',
+            url: '/index/djxw.htm',
         },
         {
             name: '学术动态',
-            url: '/index/xsdt.htm',
+            url: '/kxyj/xsdt.htm',
         },
         {
-            name: '本科生教育',
-            url: '/index/bksjy.htm',
+            name: '本科生培养',
+            url: '/index/bkspy.htm',
         },
         {
             name: '研究生教育',
             url: '/index/yjsjy.htm',
         },
-        {
-            name: '教工之家',
-            url: '/index/jgzj.htm',
-        },
     ];
 
     let type = ctx.params && ctx.params.type;
-    if (type === undefined) {
+    // check if the type is valid(is number and within range from 0 to typeUrl.length), if not, set it to 0
+    if (type === undefined || isNaN(type) || type < 0 || type >= typeUrl.length) {
         type = 0;
     } else {
-        type = Number(type);
-        if (type >= typeUrl.length) {
-            return false;
-        }
+        type = parseInt(type);
```

Comment:
The type of path variable will be either `String` or `undefined`. It won't be a `Number`.
docs: https://docs.rsshub.app/joinus/new-rss/before-start.html